images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -0,0 +1,76 @@
+################################################################################
+##  File:  Install-Msys2.ps1
+##  Desc:  Install Msys2 and 64-bit gcc, cmake, & llvm (32-bit commented out)
+################################################################################
+
+# References
+# https://github.com/msys2/MINGW-packages/blob/master/azure-pipelines.yml
+# https://packages.msys2.org/group/
+
+$origPath = $env:PATH
+$gitPath  = "$env:ProgramFiles\Git"
+
+# get info from https://sourceforge.net/projects/msys2/files/Base/x86_64/
+$msys2Uri  = "http://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20190524.tar.xz"
```

Comment:
A more recent release is available at https://github.com/msys2/msys2-installer/releases. Nightly releases are available too. It might be worth updating.

At the same time, since everything is updated anyway, it would be desirable to retrieve the non-base package.